ui/src/components/HookForm/index.jsx

```diff
@@ -358,6 +358,13 @@ export default class HookForm extends Component {
       triggerSchemaValidJson
     );
   };
+  isHookValuesSaved = () => {
```

Comment:
Instead of `isHookValuesSaved`, we could simply used `isHookDirty` directly in the `render` method.

Reply:
Could you please explain this ?

Reply:
From https://github.com/helfi92/taskcluster/commit/e16ae414dccbb983cc89a01cdd196c37038e3030, it is explained in [line 417](https://github.com/helfi92/taskcluster/commit/e16ae414dccbb983cc89a01cdd196c37038e3030#diff-6be593b857389d964b6f77d9ff45a3a8R417). Let me know if it's still confusing to understand.

Reply:
Yes that is clear, Thanks